Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -172,15 +191,24 @@ def find_zombie_processes():
     return zombies, ps_out
 
 
-def quit_driver_and_reap_children(driver):
+def quit_driver_and_display_and_reap_children(driver, display):
     """
-    Quits the driver's session and reaps all of zombie child processes
-    :param driver: The driver
+    Quits the driver's and display's sessions and reaps all of zombie child processes
+
+    :param driver: The driver session.
+    :param display: The display session.
+
     :return: None
     """
     demisto.debug(f'Quitting driver session: {driver.session_id}')
-    driver.quit()
+
     try:
+        if driver:
+            driver.quit()
+
+        if display:
```

Comment:
put in try/except and log if failed. We want to reach the zombie process stage even if driver.quite or display.stop raise an exception